lib/lita/handlers/icons.rb

```diff
@@ -0,0 +1,11 @@
+ICONS = {
```

Comment:
A note that the Zerocater API actually includes a few more labels than the ones below; I have curated the ones that I think cover the most common dietary concerns, and avoid confusion.